assets/helpers/stripeCheckout/stripeCheckout.js

```diff
@@ -31,30 +27,32 @@ const loadStripe = () => new Promise((resolve) => {
 
 });
 
-export const setup = (
-  state: StripeState,
+export const setupStripeCheckout = (
+  callback: Function,
+  closeHandler: ?Function,
+  currency: string,
   isTestUser: boolean,
-  token: Function,
-  closed: Function,
-): Promise<void> => loadStripe().then(() => {
+): Promise<*> => loadStripe().then(() => {
+
+  const handleToken = (token) => {
+    callback(token.id);
+  };
+  const defaultCloseHandler: Function = () => {};
 
   stripeHandler = window.StripeCheckout.configure({
     name: 'Guardian',
     description: 'Please enter your card details.',
     allowRememberMe: false,
-    key: isTestUser ? window.guardian.stripeKey.uat : window.guardian.stripeKey.default,
+    key: (isTestUser || false) ? window.guardian.stripeKey.uat : window.guardian.stripeKey.default,
```

Comment:
isTestUser is a boolean so cannot be undefined.  What does the `|| false` do?

Reply:
Yeap, I am catching a nonexistent case. I will remove it. Good catch 👍